src/game/features/self/NoSpread.cpp

```diff
@@ -0,0 +1,20 @@
+#include "core/commands/LoopedCommand.hpp"
+#include "game/features/Features.hpp"
+#include "game/rdr/Enums.hpp"
+#include "game/rdr/Natives.hpp"
+
+namespace YimMenu::Features
+{
+	//WEAPON::_SET_PED_INFINITE_AMMO_CLIP Changes too many things to be considered, such as granting all ammo types.
```

Comment:
This looks like an artifact from another command.

Reply:
Infinite clip, its what nospread was based from but that commented line is unneeded